_data/chains/eip155-2481632.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Recall Testnet",
+  "chain": "Recall Testnet",
+  "rpc": [
+    " https://evm.v013.node-0.testnet.recall.network/"
```

Comment:
are these updated? do we want to specify something more "permanent"?

Reply:
yes updated.